index.js

```diff
@@ -241,7 +241,7 @@ module.exports = function makeHyperFetch (opts = {}) {
             return {
               statusCode: 200,
               headers: responseHeaders,
-              data: intoAsyncIterable(`${tagVersion}`)
+              data: intoAsyncIterable(canonical)
```

Comment:
```suggestion
              data: intoAsyncIterable(`${tagVersion}`)
```

This actually shouldn't be changed because it's not for file uploading, but for tag creation (it's therefore useful to know what version the tag was created for)